drivers/cc110x/cc110x_netdev.c

```diff
@@ -353,13 +353,25 @@ static int cc110x_init(netdev_t *netdev)
         gpio_irq_disable(dev->params.gdo2);
         DEBUG("[cc110x] netdev_driver_t::init(): cc110x_apply_config() "
               "failed\n");
-        /* Pass through received error code  */
+        /* Pass through received error code */
         return retval;
     }
     else {
-        DEBUG("[cc110x] netdev_driver_t::init(): Success\n");
```

Comment:
Now that the else clause is empty, it should be removed.

Reply:
Done.

I wonder why I have put the debug message in the else case and not moved it before the `return 0;` below in the first place...